### ISSUE TITLE
nanominer 1.22 & ethash: parameters broken

### ISSUE THREAD

**Issue by UselessGuru** [nanopool/nanominer#19]
ini file:

```
; MPM autogenerated config file
checkForUpdates = false
port = 0
noLog = true
webPort = 4000

[Ethash]
devices = 0
pool1 = daggerhashimoto.eu.nicehash.com:3353
wallet = *************************************.multipoolminer
```

Note: Only device 0 should be used, ethman should not be running.

Result:

Version 1.2.2
Detected 2 devices
GPU 0 PCI 01:00.0, Platform: OpenCL, Name: Radeon RX 560 Series (Ellesmere), 8192 MB available
GPU 1 PCI 08:00.0, Platform: OpenCL, Name: Radeon RX 560 Series (Baffin), 2048 MB available
2019-04-25 18:58:19: **Started EthMan server on port 3333 without a password (read-only mode).**
2019-04-25 18:58:19: Started web interface on port 4000 without a password.
-------------------Ethash:------------------
Currency: Ethash-unknown
Wallet: *************************************.multipoolminer
Rig name: Grumpy_9ae1149b
Email was not set.
Hashrate threshold to restart miner is not set
Added pool: daggerhashimoto.eu.nicehash.com:3353
Auto pool sorting is disabled
**Using all GPU devices**
-------------Common parameters:-------------
Log path: ./logs/
Log is not written to file
GPU watchdog is enabled
Never calling reboot.bat (always restarting the miner in case of errors).

**Comment by Grumpy-Dwarf** [comment]
Thanks for reporting! Looks like it's only Ethash section has devices option broken. Will try to release bugfix before weekend. Before it is fixed workaround is to use release 1.2.1.

**Comment by Grumpy-Dwarf** [comment]
Fixed in recent [release 1.2.3](https://github.com/nanopool/nanominer/releases/tag/v1.2.3)
